README.md

```diff
@@ -154,35 +154,30 @@ and open up a web browser, and navigate to:
 ### Pushing to google cloud - Development
 
 The next step is to update the development bucket in google cloud with the new data.
-
-```shell
-gsutil -m rsync -r outputs/ gs://gtfs-data-test/report_gtfs_schedule/
-```
-
 In the case where data must be overwritten (please use caution!) a `-d` flag can be added to the above command:
```

Comment:
if above command removed, can we refactor this sentence to not reference it?

Reply:
+1

Reply:
Actually just looked at the rich diff and the command is still in, just _below_ instead of above! So swapping those 2 words oughta be enough

Reply:
I altered the other example with an optional `-d` flag so I think the new language should cover both places.